autodeconstruct.lua

```diff
@@ -123,9 +123,20 @@ local function debug_message_with_position(entity, msg)
 end
 
 function autodeconstruct.init_globals()
+  -- Find largest-range miner in the game
```

Comment:
There might be a risk that a drill that is larger gets added after initialization and so gets missed, perhaps that can be solved by updating the global when the game gets loaded

Reply:
That will be caught in `on_configuration_changed`, when the prototype list is searched again.  It fires whenever *any* mod is added, removed, or updated.

Reply:
Ah, that's fantastic